### ISSUE TITLE
What's the correct way to use 'use_git_root' and 'main_dir' to set root directory of flutter project in the action?

### ISSUE THREAD

**Issue by flyingshade** [stelynx/dart-full-coverage#10]
My flutter project is in a sub folder of $GITHUB_WORKSPACE, so based on the document, my understanding is that I need to set 'use_git_root' to false, and provide a directory name in 'main_dir'. However when I do this as below
```
- uses: stelynx/dart-full-coverage@v1.0.3
        with:
          package: test_coverage_badge
          use_git_root: false
          main_dir: test_coverage_badge

```
I receive warning that says use_git_root and main_dir are invalid inputs. And the operation failed probably due to it can't cd into the correct directory? Am I misunderstanding the document or the error?

```
Warning: Unexpected input(s) 'use_git_root', 'main_dir', valid inputs are ['package', 'ignore', 'path', 'test_file']
Run stelynx/dart-full-coverage@v1.0.3

Executing command:
cd $GITHUB_WORKSPACE && echo "// ignore_for_file: unused_import" > test/coverage_helper_test.dart && find lib -name '*.dart' -and -not -name '' | cut -c4- | awk '{printf "import '\''package:test_coverage_badge%s'\'';\n", $1}' >> test/coverage_helper_test.dart && echo "void main(){}" >> test/coverage_helper_test.dart && echo "
Processing completed. Output file in test/coverage_helper_test.dart:
" && cat test/coverage_helper_test.dart
Error: Command failed: cd $GITHUB_WORKSPACE && echo "// ignore_for_file: unused_import" > test/coverage_helper_test.dart && find lib -name '*.dart' -and -not -name '' | cut -c4- | awk '{printf "import '\''package:test_coverage_badge%s'\'';\n", $1}' >> test/coverage_helper_test.dart && echo "void main(){}" >> test/coverage_helper_test.dart && echo "
Processing completed. Output file in test/coverage_helper_test.dart:
" && cat test/coverage_helper_test.dart
/bin/sh: 1: cannot create test/coverage_helper_test.dart: Directory nonexistent


Error: /bin/sh: 1: cannot create test/coverage_helper_test.dart: Directory nonexistent
```

**Comment by campovski** [comment]
Hey, thanks for filing this issue.

I think I know what the problem is. The issue is that for some reason the latest version was not published, namely 1.0.4.

**Comment by campovski** [comment]
The last version was published to GitHub Marketplace. Please try
```
- uses: stelynx/dart-full-coverage@v1.0.4
```

**Comment by flyingshade** [comment]
It seems the issue still exist with 1.0.4... 
![image](https://user-images.githubusercontent.com/1466038/114739362-6a181c00-9d0e-11eb-9561-a051488914ae.png)

**Comment by campovski** [comment]
Interesting, I will have to look into it, but tomorrow. Hope it is ok.

**Comment by flyingshade** [comment]
> Interesting, I will have to look into it, but tomorrow. Hope it is ok.

Sure. no worries. Thanks!

**Comment by campovski** [comment]
It took me quite some time to figure out that the published script was not rebuilt (TS -> JS). We just released 1.0.5 so please try with that. Now it should work.

**Comment by campovski** [comment]
I assume this has been fixed for you and am therefore closing the issue.
